libtransmission/tr-popcount.h

```diff
@@ -0,0 +1,213 @@
+// This file Copyright © 2008-2022 Mnemosyne LLC.
+// It may be used under GPLv2 (SPDX: GPL-2.0-only), GPLv3 (SPDX: GPL-3.0-only),
+// or any future license endorsed by Mnemosyne LLC.
+// License text can be found in the licenses/ folder.
```

Comment:
If this code borrows any pieces from libpopcnt, those pieces should retain their original copyright message

For the new wrapper pieces, remove the `2008-`

Reply:
It doesn't borrow anything. Will fixup 2008-2022.

Reply:
What is the expected size of the bitfield btw? If we are expecting large bitfields could make the code significantly faster. I made more original patch so there would be no regressions but even a simple unrolling of the loop to better hide the latency gets significant gain (although costs a bit for small sizes)

Reply:
Fixed in V2.